InvenTree/templates/InvenTree/settings/plugin_settings.html

```diff
@@ -0,0 +1,112 @@
+                <td>{{ plugin.pub_date }}{% include "clip.html" %}</td>
+            </tr>
+            {% if plugin.version %}
+            <tr>
+                <td><span class='fas fa-hashtag'></span></td>
+                <td>{% trans "Version" %}</td>
+                <td>{{ plugin.version }}{% include "clip.html" %}</td>
+            </tr>
+            {% endif %}
+            {% if plugin.website %}
+            <tr>
+                <td><span class='fas fa-globe'></span></td>
+                <td>{% trans "Website" %}</td>
+                <td>{{ plugin.website }}{% include "clip.html" %}</td>
+            </tr>
+            {% endif %}
+        </table>
+
+        {% if plugin.is_package == False %}
+        <p>{% trans 'The code information is pulled from the latest git commit for this plugin. It might not reflect official version numbers or information but the actual code running.' %}</p>
+        {% endif %}
+    </div>
+    <div class="col-md-6">
+        <h4>{% trans "Package information" %}</h4>
+        <table class='table table-striped table-condensed'>
+            <col width='25'>
+            <tr>
+                <td></td>
+                <td>{% trans "Installation method" %}</td>
+                <td>
+                    {% if plugin.is_package %}
+                    {% trans "This plugin was installed as a package" %}
+                    {% else %}
+                    {% trans "This plugin was found in a local InvenTree path" %}
+                    {% endif %}
+                </td>
+            </tr>
+            {% if plugin.is_package == False %}
+            <tr>
+                <td><span class='fas fa-user'></span></td>
+                <td>{% trans "Commit Author" %}</td><td>{{ plugin.package.author }} - {{ plugin.package.mail }}{% include "clip.html" %}</td>
+            </tr>
+            <tr>
+                <td><span class='fas fa-calendar-alt'></span></td>
+                <td>{% trans "Commit Date" %}</td><td>{{ plugin.package.date }}{% include "clip.html" %}</td>
+            </tr>
+            <tr>
+                <td><span class='fas fa-code-branch'></span></td>
+                <td>{% trans "Commit Hash" %}</td><td>{{ plugin.package.hash }}{% include "clip.html" %}</td>
+            </tr>
+            <tr>
+                <td><span class='fas fa-envelope'></span></td>
+                <td>{% trans "Commit Message" %}</td><td>{{ plugin.package.message }}{% include "clip.html" %}</td>
+            </tr>
+            {% endif %}
+            <tr>
+                <td><span class='text-{{plugin.sign_color}} fas fa-check'></span></td>
+                <td>{% trans "Sign Status" %}</td>
+                <td class="bg-{{plugin.sign_color}}">{% if plugin.package.verified %}{{ plugin.package.verified }}: {% endif%}{{ plugin.sign_state.msg }}</td>
+            </tr>
+            <tr>
+                <td><span class='text-{{plugin.sign_color}} fas fa-key'></span></td>
```

Comment:
I think that the requirement for `gnupg` is not met in the docker image.  I'm getting a warning that `gpg` command is not found.

I overcame this by adding `gnupg` as a required base system package in `docker/Dockerfile` - could you please add this to the PR?

Reply:
![image](https://user-images.githubusercontent.com/10080325/141284834-27c5f2f7-d525-4007-a795-e61d3c7b90f9.png)

Now working better with gnupg installed.

What does "Cannot be checked" mean here?

Reply:
I think my key is not present in our docker image. It is a general gpg error code - schould I add a description?

Reply:
I mean, what is the mechanism for checking the keys here? Should each plugin provide a validated key? Is it checked via an external server, or locally?

Sorry, my knowledge of this is lacking compared to yours :)

Reply:
There is a PR for adding keys from github: https://github.com/inventree/InvenTree/issues/2076
The whole thing is based on gpg, which should be present with git on most systems. Checking is done locally and offline based on a public key and a signed git-commit or package. It uses the same system as git and packages in linux in general.

Reply:
Depoying keys with the signed ressources is considered bad in the industry as it would nullify the mechansism. You should check the key and add it yourself and seperate from a centerally distributed package.

Should I add a setting that disables the display of the security checks?

Reply:
> Should I add a setting that disables the display of the security checks?

I don't think that's necessary at this stage :)